lib/MetaCPAN/Web/Controller/Author.pm

```diff
@@ -58,7 +58,17 @@ sub index : Chained('root') PathPart('') Args(0) {
             = $c->model('API::Favorite')->by_user( $author->{user} )->recv;
         $took += $faves_data->{took} || 0;
 
-        $faves = [ map { $_->{fields} } @{ $faves_data->{hits}->{hits} } ];
+        my @all_fav = map { $_->{fields}->{distribution} }
+            @{ $faves_data->{hits}->{hits} };
+        my $noLatest = $c->model('API::Release')->no_latest(@all_fav)->recv;
```

Comment:
Could we call this variable `$no_latest`?  We've been avoiding snake case in general.